models/transaction_logs.go

```diff
@@ -27,19 +29,23 @@ type Execer interface {
 	Exec(string, ...interface{}) (sql.Result, error)
 }
 
+// which one to use?
 func CreateTransactionLog(db *sql.DB, log TransactionLogs) error {
-	_, err := db.Exec("INSERT INTO transaction_logs (account_num, dest_account, tran_amount, description, created_at) VALUES ($1, $2, $3, $4, $5);",
+	_, err := db.Exec("INSERT INTO transaction_logs (account_num, from_account, dest_account, tran_amount, description, created_at) VALUES ($1, $2, $3, $4, $5);",
 		log.AccountNum,
+		log.FromAccount,
 		log.DestAccount,
 		log.TranAmount,
 		log.Description,
 		log.CreatedAt)
 	return err
 }
 
-func TransactionLog(db Execer, log TransactionLogs) error {
-	_, err := db.Exec("INSERT INTO transaction_logs (account_num, dest_account, tran_amount, description, created_at) VALUES ($1, $2, $3, $4, $5);",
+// which one to use?
+func TransactioLnog(db Execer, log TransactionLogs) error {
```

Comment:
Please change back to TransactionLog, kuatirnya fungsi ini ada yg manggil terus karena namanya beda jd gabisa di run. Thank you!🙏